install/ansible/tasks/bootstrap_ubuntu.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- name: "Install required packages on Ubuntu"
+  package:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+      - unzip
+      - tar
+      - apt-transport-https
+      - btrfs-tools
+      - libapparmor-dev
+      - libseccomp-dev
+      - libseccomp2
+      - socat
+      - util-linux
```

Comment:
when: ansible_distribution == 'Ubuntu'?

Reply:
you will using this playbook only when ansible_distribution is ubuntu

Reply:
If so, we don't need the corresponding check in centos either

Reply:
done